monGARS/core/neurones.py

```
@@ -1,31 +1,196 @@
+        self,
+        exc_type: type[BaseException] | None,
+        exc: BaseException | None,
+        tb: Any,
+    ) -> None:
+        return None
 
-        async def __aexit__(
-            self, exc_type: type | None, exc: Exception | None, tb: object | None
-        ) -> None:
-            pass
+    async def run(self, *args: Any, **kwargs: Any) -> _NoOpResult:
```

Ruff (0.13.1)

Unused method argument: `args`

(ARG002)